src/stories/cartesian/brush/Brush.stories.jsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/no-default-export */
+import React from 'react';
+import BrushComponent from './BrushComponent';
+
+export default {
+  title: 'Recharts/Brush',
+  component: BrushComponent,
+};
+
+const data = [
```

Comment:
Please move the data into a shared data file. That encourages sharing datasets between stories, which makes the stories easier understandable.

Reply:
Where do you think the data files should be located? 

Looking at our test code and examples, most of the data used for each chart is different, so I think it will be less reusable in storybooks.

Reply:
In the story folder would be nice. If you look at the examples on the website, the mostly use the exact same dataset everywhere, with the 'pv' and 'uv' values.